DVSRegister/Controllers/PreRegistrationController.cs

```diff
@@ -92,6 +115,32 @@ public IActionResult SaveSponsor(SponsorViewModel sponsorViewModel)
             SummaryViewModel summaryViewModel = GetPreRegistrationSummary();
             summaryViewModel.SponsorViewModel = sponsorViewModel;
             HttpContext?.Session.Set("PreRegistrationSummary", summaryViewModel);
+
+            HttpContext?.Session.Set("IsSponsorFullNameValid", ContactViewModelValidator.ValidateFullName(sponsorViewModel.SponsorFullName));
+            HttpContext?.Session.Set("IsSponsorEmailValid", ContactViewModelValidator.ValidateEmail(sponsorViewModel.SponsorEmail));
+            HttpContext?.Session.Set("IsSponsorJobTitleValid", ContactViewModelValidator.ValidateJobTitle(sponsorViewModel.SponsorJobTitle));
+            HttpContext?.Session.Set("IsSponsorPhoneNumberValid", ContactViewModelValidator.ValidatePhoneNumber(sponsorViewModel.SponsorTelephoneNumber));
+
+            HttpContext?.Session.Set("IsFullNameValid", ContactViewModelValidator.ValidateFullName(sponsorViewModel.ContactViewModel.FullName));
+            HttpContext?.Session.Set("IsEmailValid", ContactViewModelValidator.ValidateEmail(sponsorViewModel.ContactViewModel.Email));
+            HttpContext?.Session.Set("IsJobTitleValid", ContactViewModelValidator.ValidateJobTitle(sponsorViewModel.ContactViewModel.JobTitle));
+            HttpContext?.Session.Set("IsPhoneNumberValid", ContactViewModelValidator.ValidatePhoneNumber(sponsorViewModel.ContactViewModel.TelephoneNumber));
+
+            if (string.IsNullOrEmpty(HttpContext?.Session.GetString("IsFirstVisit")))
+            {
+                HttpContext?.Session.SetString("IsFirstVisit", "true");
+            }
+            else
+            {
+                HttpContext?.Session.SetString("IsFirstVisit", "false");
+            }
+
+            if (!(bool)(HttpContext?.Session.Get<bool>("IsSponsorFullNameValid")) || !(bool)(HttpContext?.Session.Get<bool>("IsSponsorEmailValid")) || !(bool)(HttpContext?.Session.Get<bool>("IsSponsorJobTitleValid")) || !(bool)(HttpContext?.Session.Get<bool>("IsSponsorPhoneNumberValid"))
+                || !(bool)(HttpContext?.Session.Get<bool>("IsFullNameValid")) || !(bool)(HttpContext?.Session.Get<bool>("IsEmailValid")) || !(bool)(HttpContext?.Session.Get<bool>("IsJobTitleValid")) || !(bool)(HttpContext?.Session.Get<bool>("IsPhoneNumberValid")))
+            {
```

Comment:
After modifying the Validator, call method  ContactViewModelValidator(HttpContext,sponsorViewModel)

Reply:
Done